packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -1136,6 +1137,8 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
       style: inversionStyle
         ? [inversionStyle, this.props.style]
         : this.props.style,
+      isInvertedVirtualizedList:
```

Comment:
Consider forwarding this always, in case the host platform wants to do anything with the information. E.g. think @rozele was wanting to do things here with RN on desktop.